docs/rpc.md

```diff
@@ -238,6 +238,19 @@ block. It can be removed in future versions, but at the moment you can use it
 to see how much GAS is burned with a particular block (because system fees are
 burned).
 
+#### `getrawnotarypool` and `getrawnotarytransaction` calls
+
+The `getrawnotarypool` method provide the ability to retrieve a list of hashes
+for the current notary transactions. The returned structure consists of `Hashes`.
+The `Hashes`field is a map where the key is the hash of the main transaction,
+and the value is a list of related fallback transaction hashes.
```

Comment:
Don't need this precise description here, the structure itself can be changed at any moment, and thus we have the documentation for `result.RawNotaryPool`.

Reply:
Fixed.